main.js

```diff
@@ -0,0 +1,74 @@
+// Tracks Board State //
+let gameBoardObj = {
+  1: [5, 4, 3, 2, 1],
+  2: [],
+  3: []
+}
+
+
+
+
+// Handles Opening Display of Game Board //
+let renderGame = function() {Object.values(gameBoardObj).map((game) => {
+  const pegs = game.toString()
+  let gameString = `--- ${pegs}`
+  
+  let gameDisplay = gameString.split(",").join(" ")
+  
+  
+  console.log(gameDisplay)
+  
+})}
+
+// Renders Game Board in Console //
+renderGame()
+
+// Checks For Win Condition and Resets the Game //
+let winState = function() {
+
+  const fullPeg = [5, 4, 3, 2, 1]
+
+  const pegTwoFull = fullPeg.every((disc) => {
+    return gameBoardObj[2].includes(disc)
+  })
+  
+  const pegThreeFull = fullPeg.every((disc) => {
+    return gameBoardObj[3].includes(disc)
+  })
```

Comment:
DRY, a very important coding principle. Don't repeat yourself. 
You can extract this to a function and pass the object and the index you need to check.

Reply:
can also be simplified like this
```
const isWinningPeg = peg => JSON.stringify(gameBoardObj[peg]) === JSON.stringify(fullPeg);

  if (isWinningPeg(2) || isWinningPeg(3))
```